<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IgusConfig</class>
 <widget class="QWidget" name="IgusConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>433</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QDoubleSpinBox" name="spinArmLen1">
     <property name="decimals">
      <number>6</number>
     </property>
     <property name="minimum">
      <double>-1000000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1000000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>armLength[3]</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>armLength[1]</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QDoubleSpinBox" name="spinArmLen3">
     <property name="decimals">
      <number>6</number>
     </property>
     <property name="minimum">
      <double>-1000000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1000000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>offset[0]</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QDoubleSpinBox" name="spinArmLen2">
     <property name="decimals">
      <number>6</number>
     </property>
     <property name="minimum">
      <double>-1000000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1000000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QDoubleSpinBox" name="spinOffset0">
     <property name="decimals">
      <number>6</number>
     </property>
     <property name="minimum">
      <double>-1000000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1000000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>offset[1]</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>armLength[2]</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QDoubleSpinBox" name="spinOffset1">
     <property name="decimals">
      <number>6</number>
     </property>
     <property name="minimum">
      <double>-1000000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1000000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QDoubleSpinBox" name="spinArmLen0">
     <property name="decimals">
      <number>6</number>
     </property>
     <property name="minimum">
      <double>-1000000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1000000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>armLength[0]</string>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="QDoubleSpinBox" name="spinP02">
     <property name="decimals">
      <number>6</number>
     </property>
     <property name="minimum">
      <double>-1000000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1000000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="13" column="1">
    <widget class="QDoubleSpinBox" name="spinScal">
     <property name="decimals">
      <number>6</number>
     </property>
     <property name="minimum">
      <double>-1000000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1000000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label_12">
     <property name="text">
      <string>posLim[0]</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>P0[1]</string>
     </property>
    </widget>
   </item>
   <item row="14" column="1">
    <widget class="QDoubleSpinBox" name="spinvM">
     <property name="decimals">
      <number>6</number>
     </property>
     <property name="minimum">
      <double>-1000000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1000000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QDoubleSpinBox" name="spinP01">
     <property name="decimals">
      <number>6</number>
     </property>
     <property name="minimum">
      <double>-1000000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1000000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>P0[2]</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0" colspan="2">
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="14" column="0">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>vM</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QDoubleSpinBox" name="spinP00">
     <property name="decimals">
      <number>6</number>
     </property>
     <property name="minimum">
      <double>-1000000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1000000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>P0[0]</string>
     </property>
    </widget>
   </item>
   <item row="13" column="0">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>scal</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="2">
    <widget class="Line" name="line_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QDoubleSpinBox" name="spinPosLim0">
     <property name="decimals">
      <number>6</number>
     </property>
     <property name="minimum">
      <double>-1000000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1000000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="label_13">
     <property name="text">
      <string>posLim[1]</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QDoubleSpinBox" name="spinPosLim1">
     <property name="decimals">
      <number>6</number>
     </property>
     <property name="minimum">
      <double>-1000000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1000000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="15" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="2">
    <widget class="QCheckBox" name="chkInvert">
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>Invert</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLabel" name="label_14">
     <property name="text">
      <string>Angle to Dist</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QDoubleSpinBox" name="spinAngleToDist">
     <property name="decimals">
      <number>6</number>
     </property>
     <property name="minimum">
      <double>0.000001000000000</double>
     </property>
     <property name="maximum">
      <double>1000000.000000000000000</double>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
